integration/integration_test.go

```diff
@@ -0,0 +1,463 @@
+		},
+		waitUntilTestPodReady,
+		deleteTestPod,
+		execsnoopCmd, // Stop it.
+	}
+
+	runCommands(commands, t)
+}
+
+func TestMountsnoop(t *testing.T) {
+	mountsnoopCmd := &command{
+		name:           "Start mountsnoop gadget",
+		cmd:            "$KUBECTL_GADGET mountsnoop -n test-ns",
+		expectedRegexp: `test-pod\s+test-pod\s+mount.*mount\("/mnt", "/mnt", .*\) = -ENOENT`,
+		startAndStop:   true,
+	}
+
+	commands := []*command{
+		mountsnoopCmd, // Start it.
+		{
+			name:           "Run pod which tries to mount a directory",
+			cmd:            "kubectl run --restart=Never --image=busybox -n test-ns test-pod -- sh -c 'while true; do mount /mnt /mnt; done'",
+			expectedRegexp: "pod/test-pod created",
+		},
+		waitUntilTestPodReady,
+		deleteTestPod,
+		mountsnoopCmd, // Stop it.
+	}
+
+	runCommands(commands, t)
+}
+
+func TestNetworkpolicy(t *testing.T) {
+	commands := []*command{
+		{
+			name:           "Run test pod",
+			cmd:            "kubectl run --restart=Never --image=busybox -n test-ns test-pod -- sh -c 'while true; do wget -q -O /dev/null https://kinvolk.io; done'",
+			expectedRegexp: "pod/test-pod created",
+		},
+		waitUntilTestPodReady,
+		{
+			name:           "Run network-policy gadget",
+			cmd:            "$KUBECTL_GADGET network-policy monitor -n test-ns --output ./networktrace.log & sleep 15; kill $!; head networktrace.log",
+			expectedRegexp: `"type":"connect".*"test-ns".*"test-pod"`,
+		},
+		deleteTestPod,
+	}
+
+	runCommands(commands, t)
+}
+
+func TestOpensnoop(t *testing.T) {
+	opensnoopCmd := &command{
+		name:           "Start opensnoop gadget",
+		cmd:            "$KUBECTL_GADGET opensnoop -n test-ns",
+		expectedRegexp: `test-ns\s+test-pod\s+test-pod\s+\d+\s+whoami\s+3`,
+		startAndStop:   true,
+	}
+
+	commands := []*command{
+		opensnoopCmd, // Start it.
+		{
+			name:           "Run pod which calls open()",
+			cmd:            "kubectl run --restart=Never --image=busybox -n test-ns test-pod -- sh -c 'while true; do whoami; done'",
+			expectedRegexp: "pod/test-pod created",
+		},
+		waitUntilTestPodReady,
+		deleteTestPod,
+		opensnoopCmd, // Stop it.
+	}
+
+	runCommands(commands, t)
+}
+
+func TestProcessCollector(t *testing.T) {
+	if *githubCI {
+		t.Skip("Cannot run process-collector within GitHub CI.")
+	}
+
+	commands := []*command{
+		{
+			name:           "Run nginx pod",
+			cmd:            "kubectl run --restart=Never --image=nginx -n test-ns test-pod",
+			expectedRegexp: "pod/test-pod created",
+		},
+		waitUntilTestPodReady,
+		{
+			name:           "Run process-collector gadget",
+			cmd:            "$KUBECTL_GADGET process-collector -n test-ns",
+			expectedRegexp: `test-ns\s+test-pod\s+test-pod\s+nginx\s+\d+`,
+		},
+		deleteTestPod,
+	}
+
+	runCommands(commands, t)
+}
+
+func TestProfile(t *testing.T) {
+	profileCmd := &command{
+		name:           "Start profile gadget",
+		cmd:            "$KUBECTL_GADGET profile -n test-ns -p test-pod -K",
+		expectedRegexp: `cat`,
```

Comment:
```suggestion
		expectedRegexp: "cat",
```

Will it work the same?